settings.py

```diff
@@ -6,11 +6,26 @@
 values from localbuildsettings.py (if exists).
 """
 
+import time
+
+_build_date = None
+_build_timestamp = None
+
+
+def generate_timestamps():
+    """Generate build date and timestamp in desired formats."""
+    global _build_date, _build_timestamp
+
+    utc = time.gmtime()
+    _build_date = time.strftime('%Y-%m-%d-%H%M%S', utc)
+    _build_timestamp = time.strftime('%Y%m%d.%H%M%S', utc)
+
+    return _build_date, _build_timestamp
```

Comment:
Is the return needed now since all is "just" saved to the globals?

Reply:
No, you're right